pdc/apps/compose/lib.py

```diff
@@ -55,6 +56,9 @@ def insert_compose_rpms_if_nonexist(compose_rpms_in_db, cursor,
                                       sigkey_id,
                                       path_id)
         compose_rpms_in_db.add(key)
+        inserted = True
+
+    return inserted
```

Comment:
How about here return False and line 59 return True, then don't have to use inserted.